index.html

```diff
@@ -16,52 +17,79 @@
 
 <body>
 
-  
-     
+  <div class="navigation">
+    <h1>Fun Bus</h1>
```

Comment:
If you add the class container in front of the class navigation it will apply the container styles to the navigation div as well and you won't need the padding or set the widths on the ```<h1>``` and the ```<nav>``` tags. For example
```<div class="container navigation">```.